api/src/test/java/com/epam/pipeline/test/creator/docker/DockerCreatorUtils.java

```diff
@@ -0,0 +1,60 @@
+/*
```

Comment:
Tool version creation method is located in `docker` creator utils but it seems better to put it closer to tools.

Probably we can get rid or `tool` and `scan` creator utils because it won't be quite clear where to put specific methods and fields in the future as well. Only a single `docker` creator utils will be enough from my point of view.